src/lib/conversion/rotation.h

```diff
@@ -91,8 +91,11 @@ enum Rotation : uint8_t {
 	ROTATION_ROLL_90_PITCH_68_YAW_293 = 38,
 	ROTATION_PITCH_315                = 39,
 	ROTATION_ROLL_90_PITCH_315        = 40,
+	ROTATION_CUSTOM                  = 100,
 
-	ROTATION_MAX
+	ROTATION_MAX = 42 // NOTE: Update this variable to match the number of Rotation enum members!
```

Comment:
This gets a bit weird with things like mag cal auto rotation iteration through enum Rotation. What about just keeping `ROTATION_MAX = 41` right after that last valid rotation enum that isn't custom?

Reply:
Ahhh that is a nice way to solve this problem! Thank you!